slides/cdds-worskhop.mdx

```diff
@@ -573,57 +478,62 @@ I have stickers with me. Please go grab one after the talk!
 		<span>🙄</span>
 		<span>Dropdown</span>
 		<div>
-			<Code language="html">{dropdown.html}</Code>
+			<Code language="html">
+				{require('raw-loader!./code/dropdownHTML.txt').default}
```

Comment:
Hm, why `.txt`? To avoid extra loaders?

Reply:
Exactly. It’s not possible to disable eslint-loader and these files will never pass it. Maybe one way of solving it would be to setup the Gatsby environment and use the theme instead of the standalone version of MDX-deck.